client/my-sites/earn/index.ts

```diff
@@ -22,6 +27,7 @@ export default function () {
 	page( '/earn/refer-a-friend', siteSelection, sites, makeLayout, clientRender );
 
 	// These are legacy URLs to redirect if they are present anywhere on the web.
+	page( '/earn/payments-plans/:site_id', redirectToSettings, makeLayout, clientRender );
```

Comment:
The redirect is added here to ensure we catch any other links to this page, such as those in Jetpack, that need to be updated separately.